### PR TITLE
[ACEAO-5766]: Changes for Redirect Issue

### PR DESCRIPTION
**Pull request by cb-manojpathak**
Please follow the below steps:

1. Verify the latest changes(two files) and apply the same change in `CMSPencileblue` repo. Changes will be require manual changes on `CMSPencileblue`  repo for below files:
```
node_modules/@cbtn/pencilblue/include/http/middleware/routing.js
node_modules/@cbtn/pencilblue/include/http/middleware/index.js
```
2. Run the app.
3. Verify the changes.

We have verified the changes on locally for `CMSPencilblue` repo by updating the `pencilblue` repo for [ACEAO-5766](https://careerbuilder.atlassian.net/browse/ACEAO-5766).
cc: @JoseDomingoMoraNunezCB @lyliuCB

### CODE REVIEW

File: include/http/middleware/routing.js [hunk 1]
```diff
@@ -93,7 +93,7 @@ module.exports = pb => ({
         req.pathVars = pathVars
     },
     inactiveAccessCheck: (req, res) => {
-        var inactiveSiteAccess = req.route.inactive_site_access;
+        var inactiveSiteAccess = req.route ? req.route.inactive_site_access : null;
```
Comment (JoseDomingoMoraNunezCB) [inline include/http/middleware/routing.js:96]: Please explain at what moment this logic reads the Inactive status of the associated TN. I could not find that in pencilblue or CMSPencilblue

**Review by JoseDomingoMoraNunezCB — CHANGES_REQUESTED** [review]
Left a question about the functionality

File: include/http/middleware/routing.js (hunk 1 above)
Comment (lgenet) [reply include/http/middleware/routing.js:96]: Inactive site access should only be checked when a tn is disabled in the admin console. That is either upon first creation or if the site was taken down for changes. Maintenance. Or the client did not renew. 

There’s additional logic about inactive site redirects that might cause some strange behaviors. 

Also the cause for this to be null is concerning. I’d check the collections related to site objects to see if anything is malformed. 

Hope that helps 😊

File: include/http/middleware/index.js [hunk 2]
```diff
@@ -39,8 +39,8 @@ module.exports = function(pb) {
         deriveSite,
         deriveActiveTheme,
         deriveRoute,
-        localizedRouteCheck,
         inactiveAccessCheck,
+        localizedRouteCheck,
```
Comment (lyliuCB) [inline include/http/middleware/index.js:43]: why?

File: include/http/middleware/routing.js (hunk 1 above)
Comment (lyliuCB) [reply include/http/middleware/routing.js:96]: Thanks for that @lgenet

**Review by lyliuCB — CHANGES_REQUESTED** [review]
Please provide detailed test steps to demo what goal we are going to achieve by doing this change.

**Comment by cb-manojpathak** [comment]
Hello @JoseDomingoMoraNunezCB @lgenet ,
This PR is not relevant, So I am closing this for now. It was raised based on local code which was not behaved properly as I verified with @lyliuCB. 
Apologies for inconvenience!!

**Pull request closed without merging**